src/test/java/com/serenitydojo/WhenCreatingObjects.java

```diff
@@ -40,13 +40,17 @@ public void whenADogGetsFed() {
     @Test
     public void whenAnimalsPlay() {
 // TODO: Uncomment me
-//
-//        Pet fido = new Dog("Fido","Bone", 5);
-//        Pet spot = new Cat("Spot","String", 5);
-//        Pet hazel = new Hamster("Hazel", 1, "Wheel");
-//
-//        assertThat(fido.play(), equalTo("plays with bone"));
-//        assertThat(spot.play(), equalTo("plays with string"));
-//        assertThat(hazel.play(), equalTo("runs in wheel"));
+
+        Pet fido = new Dog("Fido","Bone", 5);
+        Pet spot = new Cat("Spot","String", 5);
```

Comment:
yeah, weird. we are passing `String` as toy but assert `string`
i guess it doesn't matter because the exercise is about having the common Pet superclass and implementing abstract methods. nevermind :)

Reply:
Yeah.